### DISCUSSION TITLE
tinygrad 0.5.0

### DISCUSSION THREAD

**Discussion by geohot** [tinygrad/tinygrad#656] [Announcements]
An upsetting 2223 lines of code, but so much great stuff!

* 7 backends : CLANG, CPU, CUDA, GPU, LLVM, METAL, and TORCH
* A TinyJit for speed (decorate your GPU function today)
* Support for a lot of onnx, including all the models in the backend tests
* No more MLOP convs, all HLOP (autodiff for convs)
* Improvements to shapetracker and symbolic engine
* 15% faster at running the openpilot model

<hr /><em>This discussion was created from the release <a href='https://github.com/geohot/tinygrad/releases/tag/v0.5.0'>tinygrad 0.5.0</a>.</em>

**Comment by HamzaMateen** [comment]
I will comment here just for the sake of it!

**Comment by tfmoraes** [comment]
Hi @geohot. If `extra` is not included in package how can I load onnx models?

**Reply by moracabanas** [reply]
I have this issue with `extra.datasets`

following the documentation, I follow the first example for handwritten numbers.

 I am trying to use `Poetry` to handle dependencies,
 Installing the project with `poetry add git+https://github.com/tinygrad/tinygrad.git` should be equivalent to `pip` solution.

After setting up `tinygrad` dependencies, the `extra` is not installed and I cannot access it
